deploy/olm-catalog/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml

```diff
@@ -181,8 +184,8 @@ spec:
     - name: Blog post
       url: https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator
   maintainers:
-    - email: shbose@redhat.com
-      name: Shoubhik Bose
+    - email: sbo-support@redhat.com
```

Comment:
@sbose78 who can decide what address should be used? I have seen that serverless team uses `serverless-support@redhat.com`, hence this proposal.